app/views/spree/admin/overview/_enterprises.html.haml

```diff
@@ -25,7 +25,7 @@
         %span.centered.three.columns
           = t "spree_admin_enterprises_fees"
     %div.sixteen.columns.alpha.list
-      - @enterprises.is_distributor.each do |enterprise|
+      - @enterprises.each do |enterprise|
```

Comment:
I suppose this is ok, but this view is really designed for shops (distributors) only, just adding non-shops to the view doesn't actually provide any meaningful information for those enterprises that don't sell anything. Sorry I wasn't clearer before, but I think it might be best to either a) only show shops (as you had it before), or b) change the view so that it shows information relevant to all kinds of enterprises. I can't think what this information might be (that is why there were two tabs, showing different information based on the enterprise type), so maybe we should just not show any non-shops.

I don't have a strong feeling about it though to happy with this as is if we should just get this through. I don't think it is used enough for us to stress too much about it.